plugins/org.eclipse.n4js.semver/src/org/eclipse/n4js/semver/SEMVERUtils.java

```diff
@@ -0,0 +1,227 @@
+package org.eclipse.n4js.semver;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+
+import org.eclipse.emf.common.util.EList;
+import org.eclipse.n4js.semver.SEMVERMatcher.VersionNumberRelation;
+import org.eclipse.n4js.semver.SEMVER.Qualifier;
+import org.eclipse.n4js.semver.SEMVER.QualifierTag;
+import org.eclipse.n4js.semver.SEMVER.SEMVERFactory;
+import org.eclipse.n4js.semver.SEMVER.SimpleVersion;
+import org.eclipse.n4js.semver.SEMVER.VersionComparator;
+import org.eclipse.n4js.semver.SEMVER.VersionNumber;
+import org.eclipse.n4js.semver.SEMVER.VersionPart;
+import org.eclipse.n4js.semver.SEMVER.VersionRangeConstraint;
+import org.eclipse.n4js.semver.SEMVER.VersionRangeSet;
+
+import com.google.common.base.Strings;
+
+/** Utilities to create {@link VersionNumber}s */
+public class SEMVERUtils {
+
+	static class VersionNumberDescriptor {
```

Comment:
why is this class required in addition to the stuff in `SEMVER.xcore`?
Maybe add remark to JavaDoc of class.